src/main/java/carproject/controller/CarController.java

```diff
@@ -30,7 +31,7 @@
     @Autowired
     private CarMapper carMapper;
 
-    @PostMapping("/add-new-car")
+    @PostMapping("/new")
```

Comment:
```suggestion
    @PostMapping("/")
```
ключове слово new тут не потрібно, адже нам тип POST говорить, що буде відбуватись процедура додавання нового ресурсу
https://restfulapi.net/resource-naming/